### PR TITLE
Update examples on README to allow rootless execution

### LINKED ISSUE #1413: runc run fails with "mkdir /run/runc: permission denied"
I followed the instructions found on README.md:

1. Create the filesystem: `docker export $(docker create busybox) | tar -C rootfs -xvf -`
2. Create the config: `runc spec`
3. Run the image: `runc run mycontainerid`

Got the error: `mkdir /run/runc: permission denied`

Am I missing something?

### PR DESCRIPTION
**Pull request by jwendell**
Closes #1413.

### CODE REVIEW

**Comment by crosbymichael** [comment]
Maybe a separate section for rootless instead of overwriting the default?  What do you think? 

We really need to work on our docs and getting started so thank you for the PR

**Comment by jwendell** [comment]
@crosbymichael I don't have a strong opinion on that. Thing is that currently the instructions are broken in the sense they are not saying you need to execute those commands as root. So, if we go this way of creating another section for rootless we need to fix the default section.

I'm happy to just fix the current doc adding the need of root if you prefer.

**Comment by crosbymichael** [comment]
Well its implied that you have to run them as root because that has been the status quo for many many years ;)

**Comment by cyphar** [comment]
I agree with @crosbymichael, it'd be nice if the docs split out the "normal" operation (run as root) and the new rootless mode.

**Comment by jwendell** [comment]
OK, Just pushed a new version.

@crosbymichael It was not so clear to me, that's my first contact with runc and I was strictly following the tutorial on README, and tried to execute the commands as an ordinary user.

**Comment by cyphar** [comment]
You need to include a `Signed-off-by` line in your commit. You can pass the `-s` flag to `git commit --amend` and git will include it automatically.

**Comment by jwendell** [comment]
oops, sorry. Fixed.

File: README.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ After a root filesystem is populated you just generate a spec in the format of a
 To find features and documentation for fields in the spec please refer to the [specs](https://github.com/opencontainers/runtime-spec) repository.
 
 ```bash
+# run as root
```
Comment (cyphar) [inline README.md]: Nit: You don't need to run this as root.

Comment (jwendell) [inline README.md]: That's correct. Fixed.

**Comment by cyphar** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

Thanks @jwendell

**Pull request merged**